netbox/project-static/styles/_variables.scss

```diff
@@ -1,7 +1,7 @@
 // Global variables
 
 // Set base fonts
-$font-family-sans-serif: 'Inter';
+$font-family-sans-serif: 'Plus Jakarta Sans Variable', 'Inter', system-ui, sans-serif;
```

Comment:
If we are using and including Plus Jakarta, do we still need Inter as a fallback?  And if not can we remove it from package.json as a dependency?

Reply:
We can remove it. Inter was selected as a custom font over whatever Tabler provides as a default IIRC. If we're using a different custom font, Inter can be removed.

Reply:
Done.